packages/block-library/src/archives/block.json

```diff
@@ -15,5 +15,6 @@
 	"supports": {
 		"align": true,
 		"html": false
-	}
+	},
+	"editorStyle": "wp-block-archives-editor"
```

Comment:
I don't like that this refers to handle, for me it should be the opposite. the block.json links to files and handlers are auto-registered  based on the block.json file making block.json the source of truth for block configs. Is this possible?

Reply:
Hmmm these are basically just passed-on to `register_block_type` via `register_block_type_from_metadata`. 
Ideally, `style` and `ediorStyle` would be the URLs of the stylesheets, but I don't know at this point if we can actually change it, there are lots of plugins out there that are already using them in `register_block_type`, so any changes would have to basically cascade in all functions, check if it's a URL or a handle and act accordingly :thinking: 
If we want to only change it for the `block.json` files and not make any changes to `register_block_type`, things will probably get more complicated since we have no way of dynamically defining a URL inside the static json file

Reply:
It's supported in this method  https://github.com/WordPress/wordpress-develop/blob/2a46c3987f806ea4aae642320101440260b1aba9/src/wp-includes/blocks.php#L144 that is used inside `register_block_type_from_metadata`. You can provide the local path prefixed with `file:` (similar to npm paths for local dependencies) as `style` or `editorStyle`. The rest will be handled behind the scenes. 

However, it doesn't support RTL version for CSS yet. Besides it isn't guarded with the feature flag proposed here. We discussed it in one of the comments above.

We should improve what WordPress Core has and in my opinion it would work with paths provided in `block.json`. In general, whenever adding APIs to Core we should introduce hooks more often to allow ourselves to apply such modifications inside the plugin 😅

Reply:
oooh nice, I didn't know about that :heart: 
Do you think we should do that in the plugin? Or should we do that refactor later in a separate PR?